content/docs/guides/neon-authorize-future.md

```diff
@@ -0,0 +1,53 @@
+---
+title: The future for Neon Authorize
+subtitle: Our vision for what comes next
+enableTableOfContents: true
+---
+
+[Neon Authorize](/docs/guides/neon-authorize) is an exciting feature, and we want to build more on top of it.
+
+Here's what we'd like to work on next — in no particular order (unless stated otherwise). If you see something you like, or something you'd like to see but don't, let us know in [Discord](https://discord.com/channels/1176467419317940276/1176788564890112042)!
+
+## Accepting JWTs for TCP and WebSocket connections
+
+We want Neon Authorize to work over any type of connection: TCP, HTTP, or WebSockets. This is probably our highest priority. Accepting JWTs over TCP (in the password field for the Postgres role) would mean that any Postgres SDK in any programming language could support Neon Authorize. We will build this.
+
+## Custom roles
+
+We aim to introduce custom roles for Neon Authorize, beyond the "authenticated" and "anonymous" roles we have now. We're also exploring the idea of adding "Neon-managed", which would be managed by our platform and included by default in every branch and read replica.
```

Comment:
```suggestion
We aim to introduce custom roles for Neon Authorize, in addition to the "authenticated" and "anonymous" roles we have now. We're also exploring the idea of adding a "Neon-managed" role, which would be managed by our platform and included by default in every branch and read replica.
```

Reply:
@davidgomes I am wondering if users are interested in us wanting to add a Neon-managed role. Perhaps I'm missing the value/purpose of this role from an end user's perspective.

Reply:
The idea behind Neon-managed roles is that we can guarantee they'll always exist (i.e., they can never be dropped). This is very useful for Neon Authorize in specific, because you can then guarantee that `authenticated` and `anonymous` will always exist. As it stands today, someone could accidentally delete these roles.